src/Microsoft.Sbom.Parsers.Spdx22SbomParser/Utils/SPDXToSbomFormatConverterExtensions.cs

```diff
@@ -39,8 +47,20 @@ internal static SbomFile ToSbomFile(this SPDXFile spdxFile)
     /// </summary>
     /// <param name="spdxPackage"></param>
     /// <returns></returns>
-    internal static SbomPackage ToSbomPackage(this SPDXPackage spdxPackage)
+    public static SbomPackage ToSbomPackage(this SPDXPackage spdxPackage)
     {
+        if (spdxPackage.FilesAnalyzed &&
```

Comment:
Is this a new validation? Or has it been there for a while?

Reply:
This was previously accomplished via https://github.com/microsoft/sbom-tool/blob/ef03e2b22e84f6dc7e1afd5c25bd4b7b6e674987/src/Microsoft.Sbom.Parsers.Spdx22SbomParser/Parser/SbomPackageParser.cs#L107